.github/workflows/pr.yml

```diff
@@ -7,10 +7,10 @@ jobs:
         steps:
             -   uses: actions/checkout@master
             -   name: Setup PHP, with composer and extensions
-                uses: shivammathur/setup-php@master
+                uses: shivammathur/setup-php@v2
                 with:
                     php-version: 7.2
-                    extension-csv: ctype, dom, iconv, json
+                    extension: ctype, dom, iconv, json
```

Comment:
This should be `extensions` (cf. https://github.com/shivammathur/setup-php#heavy_plus_sign-php-extension-support)

Reply:
Right you are... weird that it did not cause an error

Reply:
I'm also lucky – I read https://github.com/shivammathur/setup-php/issues/440#issuecomment-811939621 just yesterday.